Kudu.Services/Jobs/SwaggerApiDef.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Kudu.Contracts.Jobs;
+using Newtonsoft.Json;
+
+namespace Kudu.Services.Jobs
+{
+    public class SwaggerApiDef
+    {
+        [JsonProperty(PropertyName = "swagger")]
+        public String Swagger { get; set; }
```

Comment:
I think we normally use lowercase `string` for the datatype and the upper case one for the static `String` functions